impeller/docs/faq.md

```diff
@@ -100,3 +100,133 @@
     as development progresses.
 * How do you run `impeller_unittests` with Playgrounds enabled?
   * Specify the `--enable_playground` command-line option.
+* Describe Impeller in a Tweet.
+  * "Impeller is AOT mode for rendering."
+* Why did the Flutter team build Impeller?
+  * The short answer, for consistent performance, which could not be achieved
+    with Skia after years of effort by graphics experts. The longer answer...
+  * Flutter applications, via their use of Skia, were susceptible to the problem
```

Comment:
The longer form answer here reads a bit odd in bullet form. Usually bullets are separate points, but here it's just a set of paragraphs. Could it be reformatted to a single large bullet with text and line breaks?

Reply:
Yeah, @sethladd mentioned the same thing thing in https://github.com/flutter/engine/pull/54456#discussion_r1710325607. I'm going to reformat the entire doc in a later patch.